src/pages/oracles/_components/OracleGraph.tsx

```diff
@@ -265,10 +266,9 @@ function Spinner (): JSX.Element {
   )
 }
 
-async function fetchTimeFramePriceFeedInterval (api: WhaleApiClient, token: string, currency: string, timeRange: number, interval?: number): Promise<PriceFeedInterval[]> {
+async function fetchTimeFramePriceFeedInterval (api: WhaleApiClient, token: string, currency: string, timeRange: number, interval: number): Promise<PriceFeedInterval[]> {
```

Comment:
```suggestion
async function fetchTimeFramePriceFeedInterval (api: WhaleApiClient, token: string, currency: string, timeRange: number, interval: number = 15 * 60): Promise<PriceFeedInterval[]> {
```

another way to write it, @nichellekoh